.github/workflows/dockerhub-pr-with-parameters.yml

```diff
@@ -36,6 +36,9 @@ jobs:
         with:
           ref: ${{ steps.comment-branch.outputs.head_ref }}
 
+      - name: Set up Docker Buildx
```

Comment:
can you please move it between the steps:
`- name: Login to DockerHub` and 
`- name: build-tag-push` 
so that they are  grouped together

Reply:
done